Code/Framework/AzCore/AzCore/DOM/DomPrefixTree.h

```diff
@@ -133,9 +133,9 @@ namespace AZ::Dom
         //! Old node and its children at the path would be destroyed. The new subtree will be moved to the path.
         //! @param path The path which corresponds to the root node to which the subtree would be overwritten.
         //! @param subtree The subtree to move to the provided path.
-        //! @param createParentsIfNeeded Create parent entries for the provided path if the flag is on.
-        //! @return True if the subtree was overwritten successfully, false if createParentsIfNeeded is off and no target path exists.
-        bool OverwritePath(const Path& path, DomPrefixTree&& subtree, bool createParentsIfNeeded = false);
+        //! @param shouldCreateNodes Create all nodes for the provided path if the flag is on.
```

Comment:
nit: We will be creating nodes only if they are missing. Would be good to clarify that in the comments